java/ql/lib/ext/org.apache.hadoop.fs.model.yml

```diff
@@ -13,3 +13,10 @@ extensions:
       - ["org.apache.hadoop.fs", "Path", True, "Path", "(String,String)", "", "Argument[1]", "Argument[this]", "taint", "ai-manual"]
       - ["org.apache.hadoop.fs", "Path", True, "Path", "(String)", "", "Argument[0]", "Argument[this]", "taint", "ai-manual"]
       - ["org.apache.hadoop.fs", "Path", True, "Path", "(URI)", "", "Argument[0]", "Argument[this]", "taint", "ai-manual"]
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["org.apache.hadoop.fs", "FileSystem", True, "makeQualified", "(Path)", "", "Argument[0]", "path-injection", "ai-manual"]
```

Comment:
After reviewing the results in DCA, I realized that this row returns a `Path`. Which means we should follow the logic of the [incoming refactor](https://github.com/github/codeql/pull/12886) and make this a summary instead (a `Path`. could be sanitized before use).

Reply:
Makes sense! Done.